src/integration-tests/test-programs/vars_env.c

```diff
@@ -0,0 +1,23 @@
+#include <stdio.h>
+#include <stdlib.h>
+
+int main(int argc, char *argv[])
+{
+    char *path, *test1, *test2, *test3, *test4, *envtest;
+    path = getenv("PATH");
+    if(path == NULL) {
+        path = getenv("Path");
+    }
```

Comment:
This extra check isnt' necessary - envvars are [case insensitive in Windows' getenv](https://learn.microsoft.com/en-us/cpp/c-runtime-library/reference/getenv-wgetenv?view=msvc-170).

```suggestion
    path = getenv("PATH");
```